src/pathfinding_service/api.py

```diff
@@ -224,7 +224,8 @@ def process_payment(  # pylint: disable=too-many-branches
     if iou is None:
         raise exceptions.MissingIOU
 
-    log.debug("Checking IOU", iou=iou)
+    log.debug("Checking IOU", sender=iou.sender, total_amount=iou.amount)
```

Comment:
This will still be an unformatted address, you should wrap it in `to_checksum_address`.

Reply:
I'd also log the remaining information